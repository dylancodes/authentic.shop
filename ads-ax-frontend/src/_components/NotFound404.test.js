import React from 'react';
import { cleanup, render, fireEvent } from 'react-testing-library';
import 'jest-dom/extend-expect';

import Screen from './NotFound404.js';

describe('The NotFound404 Component', () => {
  test('should match and pass the screenshot test', () => {
    const { container } = render(<Screen />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
