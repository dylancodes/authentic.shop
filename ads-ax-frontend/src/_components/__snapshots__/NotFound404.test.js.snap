// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The NotFound404 Component should match and pass the screenshot test 1`] = `
<div
  styles="[object Object]"
>
  <h3>
    Sorry, page not found!
  </h3>
</div>
`;
